src/logging.md

```diff
@@ -396,6 +397,41 @@ fn run() -> Result<()> {
 # quick_main!(run);
 ```
 
+[ex-log-stdout]: #ex-log-stdout
+<a name="ex-log-stdout"></a>
+## Log to stdout instead of stderr
+
+[![log-badge]][log] [![env_logger-badge]][env_logger] [![cat-debugging-badge]][cat-debugging]
+
+Creates a custom logger configuration with [`env_logger::LogBuilder`], using [`target`] to set the target of the log output to [`env_logger::Target::Stdout`].
```

Comment:
One of the links is broken. Please test rendered content with `mdbook serve`.
Also please shorten some of the links `LogBuilder` and `Target::Stdout` would suffice. In case other examples also use this form please correct these as well for consistency.
While method `target` could use from longer `LogBuilder::target`.

Also the example description could be more concise. How about something along the lines of:

Creates a custom logger configuration using the `LogBuilder::target` to set the target of the log output to `Target::Stdout`.